src/Jackett.Common/Indexers/Definitions/CardigannIndexer.cs

```diff
@@ -1444,6 +1444,11 @@ protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuer
             variables[".Query.IsTvdbQuery"] = query.IsTvdbQuery ? "True" : null;
             variables[".Query.IsTvmazeQuery"] = query.IsTvmazeQuery ? "True" : null;
 
+            if (query.FreeTorrent)
+            {
+                variables[".Config.freeleech"] = "True";
```

Comment:
Please double check this, but I think that freeleech in all yaml config settings are set as type boolitem so the variable .Config.freeleech should be set to ".True" according to
https://github.com/Jackett/Jackett/blob/8aec74aed1c345a42b05340bb2e039a053965cf6/src/Jackett.Common/Indexers/Definitions/CardigannIndexer.cs#L270-L274

Reply:
Oh I see, yes you are right. variables[boolItem.Value ? ".True" : ".False"] resolves to variables[".True"] which is set to "True"

https://github.com/Jackett/Jackett/blob/8aec74aed1c345a42b05340bb2e039a053965cf6/src/Jackett.Common/Indexers/Definitions/CardigannIndexer.cs#L252-L258